packages/mandelbrot/assets/js/components/preview.js

```diff
@@ -60,6 +62,28 @@ export default class Preview {
         });
     }
 
+    _updateIframeRefs() {
+        this._iframe = this._el.find('[data-role="window"]');
+        this._previewIframeWindow = this._iframe ? this._iframe.get(0).contentWindow : null;
+        this._previewSize = $('[data-role="preview-size"]');
+
+        if (this._iframe) {
+            this._iframe.on('load', this._updateSize.bind(this));
```

Comment:
`.bind(this)` always creates a new function. since you do it twice here, it would be better to cache the bound method in a variable. maybe even in the constructor, since this method itself is called on an event and we don't know how many times the event will be called.

Reply:
👍